webusb/META.yml

```diff
@@ -4,3 +4,7 @@ links:
       results:
         - test: usb.serviceworker.https.html
           status: FAIL
+    - product: chrome
+      url: https://crbug.com/166689
```

Comment:
The bug says "Reason for revert: Causing flaky crashes in external/wpt/webusb/idlharness.https.any.worker.html" so this is probably not relevant.

Reply:
Reverted.